types/recharts/index.d.ts

```diff
@@ -868,6 +878,8 @@ export interface XAxisProps extends EventAttributes {
     tickSize?: number;
     interval?: AxisInterval;
     reversed?: boolean;
+    // see label section at http://recharts.org/#/en-US/api/XAxis
```

Comment:
The doc links look a bit out of place when none of the other properties have it, I'd suggest removing them and following @mthmulders 's advice of adding test cases. Thanks.